### ISSUE TITLE
Add Crust pinning service to compliance checks

### ISSUE THREAD

**Issue by SgtPooki** [ipfs-shipyard/pinning-service-compliance#154]
See https://github.com/ipfs/ipfs-webui/pull/1854

In order to add a new pinning service provider the changes should be made in the following locations:

- [ ] https://github.com/ipfs-shipyard/pinning-service-compliance/blob/main/.github/workflows/build-reports.yml
- [ ] https://github.com/ipfs-shipyard/pinning-service-compliance/blob/main/docs/index.md

**Comment by badkk** [comment]
Hey @SgtPooki , PR already been created, for the `SECRETS`:
1. For `CRUST_API_ENDPOINT`, you can use the public endpoint, which is `https://pin.crustcode.com/psa` 
2. For `CRUST_API_TOKEN`, you can follow this article to generate yourself: https://wiki.crust.network/docs/en/buildIPFSW3AuthPin#usage

**Comment by SgtPooki** [comment]
@badkk it looks crust is failing all checks. is this expected? https://ipfs-shipyard.github.io/pinning-service-compliance/pin.crustcode.com.html

**Comment by badkk** [comment]
> @badkk it looks crust is failing all checks. is this expected? https://ipfs-shipyard.github.io/pinning-service-compliance/pin.crustcode.com.html

Hey @SgtPooki I saw you merged the PR, do you still got question?

**Comment by SgtPooki** [comment]
@badkk yes, why is crust failing all of the compliance checks? Ideally, we would want to ensure crust is somewhat compliant with the pinning service spec prior to resolving https://github.com/ipfs/ipfs-webui/pull/1854

**Comment by SgtPooki** [comment]
@badkk Crust still seems to be failing all checks with `"reason": "UNAUTHORIZED",  "details": "Invalid Signature"` even though the signature was generated using previously given instructions. 

Please let us know if there is some account setup or something that needs to be done. Otherwise, we will be removing crust from the compliance checker.

**Comment by badkk** [comment]
> @badkk Crust still seems to be failing all checks with `"reason": "UNAUTHORIZED", "details": "Invalid Signature"` even though the signature was generated using previously given instructions.
> 
> Please let us know if there is some account setup or something that needs to be done. Otherwise, we will be removing crust from the compliance checker.

Would you mind to test w/this example? 
```shell
curl --location 'https://pin.crustcode.com/psa/pins' \
--header 'authorization: Bearer c3ViLWNUR01xVHdjck5FZnJ1VVV2SG1pS012MnVnSzJiemM1d2p0TmI3SkNxM0R0UG04c3Y6MHgzODU1ZDNkY2Y0OTYwYjhhMzY3MmEzYTRmNGRiYjNhMWFjMDkyNDNhODdmNDgxOTdmZmUxNTRkNDExZmUzZjQ2MDhhZDY4ODUzOGNmMTk3MWE2Zjg5NTVhZjJmOWFjYzkwYjdiYzZlZGVkMzBjNmIyZGM2NGZhZGI4ZGQ1MmM4Ng==' \
--header 'Content-Type: application/json' \
--data '{
    "cid": "QmaK2WnUQ8FPVq3gGhHZi7VuWt3ikkrCCnPcqjFUg7vQDP",
    "name": "crust-evm-arch"
}'
```
